molecule_hetznercloud/playbooks/create.yml

```diff
@@ -11,10 +11,12 @@
     ssh_user: root
     ssh_key_path: "{{ molecule_ephemeral_directory }}/ssh_key"
     ssh_key_name: "{{ resource_namespace }}"
+    ssh_key_type: "{{ lookup('ansible.builtin.env', 'MOLECULE_SSH_KEY_TYPE') | default(rsa) }}"
   tasks:
     - name: Create SSH key
       community.crypto.openssh_keypair:
         path: "{{ ssh_key_path }}"
+        type: "{{ ssh_key_type }}"
```

Comment:
Based on https://www.openssh.com/txt/release-8.7 (in the **Imminent deprecation notice** section)

I think we can safely hardcode the value to `ed25519`, as it has been supported by Openssh 6.5 which has been release about 10 years ago.

Reply:
hardcoded ed25519